Podcast/BookmarkViewController.swift

```diff
@@ -101,8 +113,18 @@ class BookmarkViewController: UIViewController, UITableViewDelegate, UITableView
     func cardTableViewCellDidPressPlayPauseButton(cardTableViewCell: CardTableViewCell) {
         guard let cardIndexPath = bookmarkTableView.indexPath(for: cardTableViewCell), let card = cards[cardIndexPath.row] as? EpisodeCard else { return }
         
-        //card.isPlaying = !card.isPlaying
-        //cardTableViewCell.setPlayButtonToState(isPlaying: card.isPlaying)
+        if cardIndexPath == currentlyPlayingIndexPath { return }
+        
+        if let indexPath = currentlyPlayingIndexPath {
+            if let cell = bookmarkTableView.cellForRow(at: indexPath) as? CardTableViewCell {
+                cell.setPlayButtonToState(isPlaying: false)
+            }
+        }
+        currentlyPlayingIndexPath = cardIndexPath
+        cardTableViewCell.setPlayButtonToState(isPlaying: true)
+        let appDelegate = UIApplication.shared.delegate as! AppDelegate
+        appDelegate.showPlayer(animated: true)
+        Player.sharedInstance.playEpisode(episode: card.episode)
```

Comment:
Do we want to be able to pause by clicking on the cell? I thought the "Now Playing" label implied that this wasn't a typical play/pause button. This is similar to how Spotify functions with cells controlling the starting of playback and the mini player being used to control play/pause/etc